ibis/backends/postgres/compiler.py

```diff
@@ -588,3 +588,26 @@ def visit_Cast(self, op, *, arg, to):
         return self.cast(arg, op.to)
 
     visit_TryCast = visit_Cast
+
+    def visit_Hash(self, op, *, arg):
+        arg_dtype = op.arg.dtype
+
+        if arg_dtype.is_int16():
+            return self.f.hashint2extended(arg, 0)
```

Comment:
Postgres `hash*extended` functions require a seed to be passed. I've just passed 0, since Ibis `hash` doesn't take a seed argument.

Reply:
Why are the `extended` versions required?

Reply:
Oh right, to get the 64 bit versions.